torchx/lib/torchx/backend.ex

```diff
@@ -717,6 +717,24 @@ defmodule Torchx.Backend do
     {p, l, u}
   end
 
+  @impl true
+  def pad(out, tensor, constant, config) do
+    config =
+      config
+      |> Enum.map(fn {a, b, c} ->
+        if a < 0 || b < 0 || c != 0, do: raise("{#{a}, #{b}, #{c}} padding is not supported.")
```

Comment:
When all values are boolean, you can use `or` as a strictly boolean operator!

```suggestion
        if a < 0 or b < 0 or c != 0 do
          raise ArgumentError, "{#{a}, #{b}, #{c}} padding is not supported"
        end

```